Add note about not using apt/dnf searches to AMLFS Client install docs (#3244)
MicrosoftDocs/azure-stack-docs: azure-managed-lustre/includes/client-install-version-ubuntu.md

> [!NOTE]
> This command installs a metapackage that will keep the version of Lustre aligned with the installed kernel. In order for this to work, you must use `apt full-upgrade` instead of simply `apt upgrade` when updating your system.

> > [!NOTE]
> > Running a `apt search amlfs-lustre-client` does not show all available packages for your distro. To see all available `amlfs-lustre-client` packages, please run `apt list -a "amlfs-lustre-client*"`.

Reviewer: ```suggestion

> [!NOTE]
> Running an `apt search amlfs-lustre-client` doesn't show all available packages for your distro. To see all available `amlfs-lustre-client` packages, run `apt list -a "amlfs-lustre-client*"`.
```
